src/routes/docs/components/alert.md

```diff
@@ -67,29 +67,30 @@ You can also include a descriptive icon to complement the message inside the ale
 ```svelte example class="flex flex-col gap-4" hideScript
 <script>
   import {Alert} from "flowbite-svelte";
+  import { InfoCircleSolid } from 'flowbite-svelte-icons'
 </script>
 <Alert>
-  <svg slot="icon" aria-hidden="true" class="w-5 h-5" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-7-4a1 1 0 11-2 0 1 1 0 012 0zM9 9a1 1 0 000 2v3a1 1 0 001 1h1a1 1 0 100-2v-3a1 1 0 00-1-1H9z" clip-rule="evenodd"></path></svg>
+  <InfoCircleSolid  class="flex-shrink-0 text-primary-800 dark:text-primary-400 inline w-4 h-4 mr-3" />
```

Comment:
Why don't you use `slot="icon"` here? If you did the only settings needed would be `size="sm"` or `class="w-4 h-4"`

Reply:
Thank you for the catch. I update them.